src/Umbraco.Web.UI.Client/src/less/property-editors.less

```diff
@@ -224,7 +224,6 @@ ul.color-picker li a  {
   max-height:100%;
   margin:auto;
   display:block;
-  background-image: url(../img/checkered-background.png);

```

Comment:
@madsrasmussen I think this is useful, when selecting a white png image - otherwise it is kinda invisible.